bl-content/databases/categories.php

```diff
@@ -0,0 +1,2 @@
+<?php defined('BLUDIT') or die('Bludit CMS.'); ?>
+{"computers":{"name":"Computers","template":"","description":"elit a mi vehicula, ac rhoncus nunc malesuada. Aliquam ac rutrum nisl, ut elementum felis. Nulla nisl mauris, lobortis ut tristique in, porttitor non ipsum. Nulla dignissim ac mauris a iaculis. Curabitur pharetra ac enim a sodales. Sed iaculis leo tincidunt efficitur eleifend. Nulla finibus pharetra laoreet. Duis eget mauris ut enim blandit sodales. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.","list":[]},"general":{"name":"General","description":"","template":"","list":[]},"music":{"name":"Music","description":"","template":"","list":[]},"videos":{"name":"Videos","description":"","template":"","list":[]}}
```

Comment:
Remove the database contents. Its not required in the main codebase. Its auto generated